packages/components/modules/navigations/web/Header/AccountMenu/AccountPopover/types.ts

```diff
@@ -2,23 +2,14 @@ import type { FC } from 'react'
 
 import type { CSSProperties } from '@mui/material/styles/createMixins'
 
-import type {
-  AddProfileMenuItemProps,
-  ProfilesListProps,
-  SwitchProfileMenuProps,
-} from '../../../../../profiles/web'
+import type { AddProfileMenuItemProps, ProfilesListProps } from '../../../../../profiles/web'
+import { CurrentProfileMenuProps } from './CurrentProfileMenu/types'
 import type { LogoutItemProps } from './LogoutItem/types'
-import type { MenuItemsProps } from './MenuItems/types'
 
-export interface AccountPopoverProps {
+export interface AccountPopoverProps
+  extends Omit<CurrentProfileMenuProps, 'handlePopoverOnClose' | 'setOpenProfilesList'> {
```

Comment:
@priscilladeroode I think we should've `Pick` the ones we want instead of `Omit`, so we don't need to worry if we ever add new props to `CurrentProfileMenu` that should not be part o `AccountPopoverProps`